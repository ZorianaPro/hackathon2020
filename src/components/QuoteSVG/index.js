import React from 'react';

const QuoteSVG = (props) => (
  <svg className="QuoteSVG" {...props}>
    <use xlinkHref="#QuoteSymbol" />
  </svg>
);

export default QuoteSVG;
