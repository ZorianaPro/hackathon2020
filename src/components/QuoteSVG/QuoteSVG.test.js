import React from 'react';
import { shallow } from 'enzyme';
import QuoteSVG from './';

describe('QuoteSVG', () => {
  it('renders without crashing', () => {
    expect(shallow.bind(shallow, <QuoteSVG />)).not.toThrow();
  });

  it('renders as expected', () => {
    expect(shallow(<QuoteSVG />)).toMatchSnapshot();
  });
});
