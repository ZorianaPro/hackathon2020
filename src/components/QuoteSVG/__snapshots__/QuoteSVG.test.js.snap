// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuoteSVG renders as expected 1`] = `
<svg
  className="QuoteSVG"
>
  <use
    xlinkHref="#QuoteSymbol"
  />
</svg>
`;
